al-jazari/roster.json

```diff
@@ -97,7 +97,11 @@
       "materials": [
         {
           "type": "cv"
-        }
+        },
+	{
+	  "type": "video",
+          "url": "https://www.youtube.com/watch?v=pfu1WzGhr3Y&feature=youtu.be"
+	}
```

Comment:
Can you please change the video object to
{
  "type": "video",
   "url": "pfu1WzGhr3Y"
 }